### ISSUE TITLE
interpro.tsv missing

### ISSUE THREAD

**Issue by chrishah** [nextgenusfs/funannotate#703]
**Are you using the latest release?**
No, I am using v1.8.7, but as far as I can see from the code in the current version the 'problem' persists.

**Describe the bug**
It's not a bug of funannotate. As part of the `funannotate setup` step it downloads `interpro.xml` and `interpro.tsv` files from EBI. The `interpro.tsv` file doesn't seem to be available on the ebi side any more, so it's not downloaded, but setup also doesn't give a warning about it. The `funannotate annotate` module expects `interpro.tsv` in the database and crashes if it's not there - line 1636 `annotate.py`
```INTERPRO = lib.iprTSV2dict(os.path.join(FUNDB, 'interpro.tsv'), IPRterms)```

I've replaced the line above with:
```INTERPRO = lib.iprxml2dict(os.path.join(FUNDB, 'interpro.xml'), IPRterms)```

It seems to work, but I wanted to doublecheck with you whether you see any issues with that fix or if you have any other suggestion on how to deal with it. Thanks!

Looking forward to your feedback!

Thanks for creating funannotate, btw - it's awesome!

Best wishes,
Christoph

**Comment by nextgenusfs** [comment]
Thanks for reporting @chrishah, that is unfortunate, the XML takes forever to parse.  In theory the data should be the same.  I need to look back at the code though, I feel like there was some specific reason I switched to TSV from XML (original).

**Comment by nextgenusfs** [comment]
well the file is still there so I can't reproduce this: http://ftp.ebi.ac.uk/pub/databases/interpro/entry.list.  Is it possible that it didn't download when you ran `funannotate setup`?  I just changed the links to http from ftp protocol, that can sometimes cause problems on some systems.

**Comment by chrishah** [comment]
Dear @nextgenusfs,

Thanks for looking into this. I have to admit, it appears that it was my bad. I realized now, that I used funannotate 1.7.4 to do setup and there it seems the entry.list file wasn't yet downloaded. 
Also, I was expecting to find a file 'interpro.tsv' on the server, because this is what funannotate 1.8.7 (which I am using now) is looking for and handn't realized that what funannotate downloads is `entry.list`, which is then renamed. Setup step with funannotate 1.8.7 gets the interpro.tsv file as expected and everything is good.

I am very sorry for false alarm and the inconvenience this may have caused!!! Thanks again for your support!

cheers,
Christoph
